Formula/ack.rb

```diff
@@ -38,7 +38,7 @@ def install
       man1.install "blib/man1/ack.1"
     else
       bin.install "ack-v#{version.to_s.tr("-", "_")}" => "ack"
-      system "pod2man", "#{bin}/ack", "ack.1"
+      system "#{Formula["pod2man"].opt_bin}/pod2man", "#{bin}/ack", "ack.1", "--release=ack v#{version}"
```

Comment:
@bo98 super weird. This still seems to be using a completely different version of `pod2man`? 

```console
$ diffoscope ack--3.5.0.arm64_big_sur.bottle.tar.gz ack--3.5.0.mojave.bottle.tar.gz
...
│ │ -.\" Automatically generated by Pod::Man 4.10 (Pod::Simple 3.35)
│ │ +.\" Automatically generated by Pod::Man 2.27 (Pod::Simple 3.28)
```

Reply:
#75722 will fix.

Reply:
`Pod::Simple` version will differ however, since that is not shipped with the `pod2man` formula. We'll need to normalise that.

We could either:

* Bundle an updated `Pod::Simple` in `pod2man` formula instead of relying on the one shipped with Perl.
* Avoid system Perl entirely.

Reply:
Using brewed `perl` for generating the manpage with `pod2man` generates a runtime dependency? Weird.

Reply:
No, but the rest of the formula would be using Perl.

Reply:
> #75722 will fix.

Good catch @bo98.

> * Bundle an updated `Pod::Simple` in `pod2man` formula instead of relying on the one shipped with Perl.

This makes sense to me.

Another option: if the `perl` formula ships `pod2man`: perhaps we should just remove the `pod2man` formula in favour of the `perl` one?



> * Avoid system Perl entirely and just use brewed Perl. Does mean an extra runtime dependency though.

I'm on the fence about this. I kinda hate whenever we have to force a brewed dependency. On the flip side, `pod2man` (if it even sticks around) is never likely to be more than a `:build` dependency on (soon to be) bottled stuff so: is that a big deal?

Reply:
> I'm on the fence about this. I kinda hate whenever we have to force a brewed dependency. On the flip side, `pod2man` (if it even sticks around) is never likely to be more than a `:build` dependency on (soon to be) bottled stuff so: is that a big deal?

`perl` is a runtime dependency. I don't think it's a good idea to mess around with using a brewed Perl at build time but a system Perl at runtime.

I'll look into bundling `Pod::Simple`.

Reply:
thanks @Bo98!